### PR TITLE
changed more screen text

### PR DESCRIPTION
**Pull request by noalecohen**

### CODE REVIEW

**Comment by noalecohen** [comment]
finished first task

File: src/AppNavigator.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import {NavigationContainer} from '@react-navigation/native';
+import React from 'react';
+import HomeIcon from './assets/icons/HomeIcon';
+import MoreIcon from './assets/icons/MoreIcon';
+import VodIcon from './assets/icons/VodIcon';
+import Colors from './constants/Colors';
+import HomeScreen from './features/Home/HomeScreen';
+import MoreScreen from './features/More/MoreScreen';
+import VodScreen from './features/Vod/VodScreen';
+//const Tab = createBottomTabNavigator();
```
Comment (BenMoveo) [inline src/AppNavigator.tsx]: please don't push commented code.

File: src/assets/icons/AccountIcon.tsx [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+  style: ViewStyle | ViewStyle[];
+}
 
-const AccountIcon = () => {
+const AccountIcon = (props: Props) => {
   return (
-    <Svg width="26" height="24" viewBox="0 0 26 24">
+    <Svg width="26" height="24" viewBox="0 0 26 24" style={props.style}>
       <G fill="none" fill-rule="evenodd">
         <G fill-rule="nonzero">
```
Comment (BenMoveo) [inline src/assets/icons/AccountIcon.tsx]: please note the SVG attributes in react becomes camelCase
i.e attributes like stroke-width and fill-rule become strokeWidth and fillRule, etc.

File: src/features/More/MoreScreen.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import {View, Text, StyleSheet} from 'react-native';
 const MoreScreen = () => {
   return (
     <View style={styles.text}>
-      <Text>More Screen</Text>
+      <Text>More Screen Test</Text>
```
Comment (BenMoveo) [inline src/features/More/MoreScreen.tsx]: what is this test ?

**Review by BenMoveo — APPROVED** [review]
Ive approved your PR.
Please address my comments (it can be with a comment or a like emoji)
After addressing my comments you can merge your PR.
On the PR homepage at the bottom, you will see a green merge button

File: src/features/More/MoreScreen.tsx (hunk 3 above)
Comment (noalecohen) [reply src/features/More/MoreScreen.tsx]: Sorry, I forgot to delete this check.
I deleted it now.

**Pull request merged**